client/login/magic-login/emailed-login-link-expired.jsx

```diff
@@ -37,11 +37,9 @@ class EmailedLoginLinkExpired extends React.Component {
 					replaceCurrentLocation={ true }
 				/>
 				<EmptyContent
-					action={ translate( 'Return to WordPress.com' ) }
-					actionCallback={ function() {
-						page( '/' );
-					} }
-					actionURL={ '/' }
+					action={ translate( 'Back to WordPress.com' ) }
+					actionCallback={ goBackToWordPressDotCom }
```

Comment:
Looking at the code for the `EmptyContent` component, `actionCallback` is not required here. The `<a>` tag will properly get its `href` value set to the absolute URL we pass below in `actionURL`. 
This will make it open the URL without using the `page()` framework.